bindings/dotnet/UnicornManaged/UnicornManaged.fsproj

```diff
@@ -1,56 +1,45 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>0c21f1c1-2725-4a46-9022-1905f85822a5</ProjectGuid>
-    <OutputType>Library</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Unfortunately, I can't grab a .net5.0 on Ubuntu 22.04. Is it possible to target .net6?

Reply:
It's possible to build the project with .NET6, but I wasn't sure if targeting would be okay, since I already upgraded the target from an older .NET framework version.